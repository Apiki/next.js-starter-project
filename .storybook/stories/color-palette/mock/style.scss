@import '~nodeModules/sass-to-js/sass/_sass-to-js.scss';
@import '@styles/base/1.settings/_1.colors.scss';
@import '@styles/base/1.settings/_1.fonts.scss';

#root:after,
#root:before {
  display: none;
}

#root:after {
  content: sassToJs( $colors );
}

#root:before {
  content: sassToJs( $fonts );
}
