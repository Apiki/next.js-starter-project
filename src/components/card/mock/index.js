import React from 'react';
import { text } from '@storybook/addon-knobs/react';

import { Card } from '../index';

export const usage = () => (
  <Card
    featuredMedia={{
      alt: 'Card Thumbnail',
      sizes: {
        medium: {
          height: 413,
          width: 309,
          url: text('URL', 'https://source.unsplash.com/413x309'),
        },
      },
    }}
    link={text('Link', '#')}
    title={text('Title', 'Card Title')}
    excerpt={text('Excerpt', 'I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.')}
  />
);
