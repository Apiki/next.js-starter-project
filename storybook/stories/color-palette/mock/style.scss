@import '~nodeModules/sass-to-js/sass/_sass-to-js.scss';

#root:after,
#root:before {
  display: none;
}

#root:after {
  content: sassToJs( $colors );
}

#root:before {
  content: sassToJs( $fonts );
}
