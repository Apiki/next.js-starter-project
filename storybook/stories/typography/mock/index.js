import { withSassData } from 'storybook-ui';

import { Typography } from 'storybook-ui';

export const Usage = withSassData(Typography, ':before');
