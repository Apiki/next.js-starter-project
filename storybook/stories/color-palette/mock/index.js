import { withSassData, ColorPalette } from 'storybook-ui';

import './style.scss';

export const Usage = withSassData(ColorPalette, ':after');
