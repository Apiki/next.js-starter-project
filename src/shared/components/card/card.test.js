import renderer from 'react-test-renderer';

import { usage } from './mock';

it('Should Card to match snapshot', () => {
  const tree = renderer.create(usage()).toJSON();
  expect(tree).toMatchSnapshot();
});
