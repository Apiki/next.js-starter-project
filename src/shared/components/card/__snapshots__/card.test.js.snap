// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Card to match snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card__image"
  >
    <img
      alt="Card Thumbnail"
      src="https://source.unsplash.com/413x309"
    />
    <span
      className="card__title"
    >
      Card Title
    </span>
  </div>
  <div
    className="card__content"
  >
    <p>
      I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.
    </p>
  </div>
  <div
    className="card__action"
  >
    <a
      className="card__link"
      href="#"
    >
      This is a link
    </a>
  </div>
</div>
`;
